src/Valitron/Validator.php

```diff
@@ -61,9 +62,7 @@ public function __construct($data, $fields = array(), $lang = null, $langDir = n
      */
     public static function lang($lang = null)
     {
-        if($lang !== null) {
-            static::$_lang = $lang;
-        }
+        $lang !== null && static::$_lang = $lang;
```

Comment:
Hello, for me, the improved line looks much more complex then it was before. Additionally you will not see if second part is covered in test coverage reports.